api-generation/build.gradle

```diff
@@ -17,8 +17,6 @@ shadowJar {
   mergeServiceFiles()
 }
 
-artifacts {
-  api shadowJar
-}
-
 build.dependsOn shadowJar
+
+defineStandardPublication().call()
```

Comment:
interesting, never seen this one before outside of SW project

Reply:
in SW, the publishing section is not defined globally. I'm not sure whether it's good or bad, but we can decide in each project whether a [regular jar](https://github.com/h2oai/sparkling-water/blob/master/gradle/publish.gradle#L68) or a [shadowed jar](https://github.com/h2oai/sparkling-water/blob/master/gradle/publish.gradle#L47) will be published.